content/docs/quickstart/_index.md

```diff
@@ -7,6 +7,30 @@ weight: 1
 hidechildren: true # this flag hides all sub pages in the sidebar-multicard.html
 ---
 
+## Keptn Hello World
+
+Run the Keptn Hello, World! example:
+
+```
+docker run --rm -it \
+--name thekindkeptn \
+-v /var/run/docker.sock:/var/run/docker.sock:ro \
+gardnera/thekindkeptn:0.0.5
```

Comment:
Would it make sense to move it to Keptn Sandbox or to Keptn Examples?

Reply:
No, IMO this is the very first thing a Keptn beginner would want so it belongs on the most visible part of the QS, not hidden in a "side" repo. What we should probably do is transfer the docker image to the official Keptn repo rather than my personal one.